src/functions-and-arrays.js

```diff
@@ -52,14 +129,41 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(arr) {
+  if (arr.length === 0) {
+    return null
+  } else {
+    //criar uma array vazia
+    let result = []
+    //rodar um loop que acesse cada elemento da array
+    for (let i = 0; i < arr.length; i++) {
+      //o indexOf() acha o indice com base na primeira ocorrencia, entao se a string for igual, o numero que ele vai apontar vai ser diferente do numero do indice i
+      if (arr.indexOf(arr[i]) === i) {
+        //pra retornar uma array sem duplicadas, vou adicionar a string na array vazia apenas se o indexOf for igual ao i
```

Comment:
continue fazendo comentários quando vc não estiver entendendo, muito bom 👏 só que quando vc estiver trabalhando provavelmente terá que removê-los quando for submeter seu código ok?